File: Easy/Java/1480.-Running-Sum-of-1d-Array.java
class Solution {
    public int[] runningSum(int[] nums) {
        int l=nums.length;
        int sum[] = new int[l];
        int c=0;
        for (int i=0;i<l;i++){
            sum[i]=c+nums[i];
            c+=nums[i];
        }   
        return sum;
    }
}

File: Easy/Python/1480.-Running-Sum-of-1d-Array.py
class Solution:
    def runningSum(self, nums: List[int]) -> List[int]:
        n=len(nums)
        runningSum=[]
        for i in range(n):
            runningSum.append(sum(nums[:i+1]))
        return runningSum
# OR
class Solution:
    def runningSum(self, nums: List[int]) -> List[int]:
        runningSum=[]
        c=0
        for i in nums:
            runningSum.append(c+i)
            c+=i
        return runningSum
            